include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -30,7 +30,7 @@
 *
 * This value is used by some API functions to behave as this version of the header expects.
 */
-#define ORT_API_VERSION 13
+#define ORT_API_VERSION 14
```

Comment:
Need to bump version up to 14 (current development version) because variadic IO for custom ops is only supported starting now.